app/models/order.rb

```diff
@@ -9,6 +9,7 @@ class Order < ActiveRecord::Base
   belongs_to :stockit_organisation
   belongs_to :organisation
   belongs_to :beneficiary
+  belongs_to :address
```

Comment:
Do you mean referencing the order from the Address ?
I think we'd like to avoid making any architectural changes right now. We're just adding a reference to link them together for clarity/simplicity. Technically the address belongs to the contact